reference/Chronicler2.v0.yaml

```diff
@@ -138,102 +277,166 @@ components:
+                        ballsNeeded:
+                          type: integer
+                        baserunners:
+                          type: array
+                          items:
+                            type: object
+                        batter:
                           type: 'null'
-                        x:
+                        homeScore:
                           type: integer
-                        'y':
+                        inning:
                           type: integer
-                  rosterSlots:
-                    type: array
-                    items:
-                      type: object
-                      properties:
-                        active:
+                        outs:
+                          type: integer
+                        outsNeeded:
+                          type: integer
+                        pitcher:
+                          type: 'null'
+                        shame:
                           type: boolean
-                        location:
+                        step:
+                          type: integer
+                        strikes:
+                          type: integer
+                        strikesNeeded:
+                          type: integer
+                        teamAtBat:
                           type: string
-                        orderIndex:
+                        topOfInning:
+                          type: boolean
+                        totalBases:
                           type: integer
-                  team:
+                  gameWinnerId:
+                    type: string
+                  homePitcher:
+                    type: object
+                    properties:
+                      id:
+                        type: string
+                      name:
+                        type: string
+                  homeTeam:
                     type: object
                     properties:
                       emoji:
                         type: string
+                      gameWinner:
+                        type: boolean
                       id:
                         type: string
                       name:
                         type: string
+                      nickname:
+                        type: string
                       primaryColor:
                         type: string
                       secondaryColor:
                         type: string
+                      shorthand:
+                        type: string
+                      slogan:
+                        type: string
+                  id:
+                    type: string
+                  numberInSeries:
+                    type: integer
+                  seasonId:
+                    type: string
+                  seriesLength:
+                    type: integer
+                  startTime:
+                    type: string
+                  started:
+                    type: boolean
```

Comment:
is there a way of putting comments here? about how games which have completed might not necessarily have started (at least, in the current data)

Reply:
Added a description for this. That's... a really weird case huh. Wonder what causes it.

Reply:
yeah, it's very strange. I've left a big fat "problem for future me" comment related to that in the reblase code